watchtower-web/src/main/java/com/tracelink/appsec/watchtower/web/controller/LoginController.java

```diff
@@ -15,15 +17,23 @@
 
 	private final ClientRegistrationRepository clientRegistrationRepository;
 
-	public LoginController(@Autowired(required = false) ClientRegistrationRepository clientRegistrationRepository) {
+	private final boolean allowRegistration;
+
+	public LoginController(
+			@Autowired(required = false) ClientRegistrationRepository clientRegistrationRepository,
+			@Value(value = "${watchtower.allowRegistration?:true}") boolean allowRegistration) {
```

Comment:
I think there's some inconsistency with the name of this property. In this annotation it is `allowRegistration`, but in other places, you have `allowedRegistration` (e.g. `deploy.yaml` and `README.md`). 
Also, you have a `?` after the application property name in this class. Should there be a `?` after `watchtower.runAfterStartup` in the PRScanningService and UploadScanningService as well?

Reply:
I chose to leave the runAfterStartup as-is because the @value annotations have default values, for the conditional in the controller, I can't default something if it doesn't exist.